src/index.html

```diff
@@ -5,9 +5,54 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-    <link rel="stylesheet" href="./css/style.css">
-
     <title>Hakim Portfolio</title>
+
+    <!-- Global styles -->
+    <link rel="stylesheet" href="./css/reset.css" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <link rel="stylesheet" href="./css/home.css" />
+    <!-- components -->
+    <link rel="stylesheet" href="./css/components/toolbar.css" />
+    <link rel="stylesheet" href="./css/components/headline.css" />
+    <!-- google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body></body>
+  <body>
+    <header class="header">
+      <nav class="toolbar">
+        <div class="toolbar__logo-wrapper">
+          <span class="toolbar__logo">Hakim</span>
+        </div>
+        <button class="toolbar__menu-button" type="button">
+          <img src="./assets/icons/menu.svg" alt="Menu icon" />
+        </button>
+      </nav>
+      <div class="headline">
+        <h1 class="headline__title">I’m Hakim Glad to see you!</h1>
+        <p class="headline__text">
+          I’m a software developer! I can help you build a product, feature or
+          website. Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <a class="headline__cta" href="#">LET’S CONNECT</a>
+        <div class="headline__social-links">
+          <a>
+            <img src="./assets/icons/twitter.svg" alt="" />
+          </a>
+          <a>
+            <img src="./assets/icons/linkedin.svg" alt="" />
+          </a>
+          <a>
+            <img src="./assets/icons/github.svg" alt="" />
+          </a>
+        </div>
+      </div>
+    </header>
```

Comment:
- [ ] Please the rest of the page content should be in the body but not in the header. Please kindly move it out of the header and change the `div` to `section`. By doing so your code will look more professional and more semantic. lets work like experts 😃